package.json

```diff
@@ -34,25 +34,30 @@
     }
   },
   "dependencies": {
-    "@itwin/core-bentley": "3.7.13",
-    "@itwin/core-common": "3.7.13",
-    "@itwin/core-frontend": "3.7.13",
-    "@itwin/core-geometry": "3.7.13",
-    "@itwin/core-i18n": "3.7.13",
-    "@itwin/core-markup": "3.7.13",
-    "@itwin/core-orbitgt": "3.7.13",
-    "@itwin/core-quantity": "3.7.13",
-    "@itwin/presentation-common": "3.7.13",
-    "@itwin/presentation-frontend": "3.7.13",
-    "@itwin/webgl-compatibility": "3.7.13",
+    "@itwin/core-bentley": "4.1.0",
+    "@itwin/core-common": "4.1.0",
+    "@itwin/core-frontend": "4.1.0",
+    "@itwin/core-geometry": "4.1.0",
+    "@itwin/core-i18n": "4.1.0",
+    "@itwin/core-markup": "4.1.0",
+    "@itwin/core-orbitgt": "4.1.0",
+    "@itwin/core-quantity": "4.1.0",
+    "@itwin/presentation-common": "4.1.0",
+    "@itwin/presentation-frontend": "4.1.0",
+    "@itwin/webgl-compatibility": "4.1.0",
     "base64-js": "^1.5.1"
   },
   "devDependencies": {
-    "@itwin/eslint-plugin": "3.7.13",
-    "@typescript-eslint/eslint-plugin": "^4.33.0",
-    "@typescript-eslint/parser": "^4.33.0",
+    "@itwin/eslint-plugin": "4.0.0-dev.33",
```

Comment:
Is this updated version necessary? If so, should we request (or create) a non-dev 4.0 release of @itwin/eslint-plugin?

Reply:
As far as I can tell, @itwin/eslint-plugin has been abandoned, but I haven't yet figured out what (if anything) replaces it. Since all it affects is building, the fact that it is a dev release shouldn't hurt anything for now.

Reply:
it has not been abandoned, but it desparately needs a non-dev release

Reply:
Is a non-dev release planned soon? Is there anything holding it back?

Reply:
> it has not been abandoned, but it desperately needs a non-dev release

Good to know. As an aside, the reason I'm using this version is that it works, and when I tried the latest dev release (4.0.0-dev.48), I got an error about a missing plugin. Since I was in a hurry to get a 4.1-based release out and this doesn't affect the end result of the build, I went with the version that I already knew worked for me.

Reply:
@aruniverse any reason we keep publishing dev versions of `@itwin/eslint-plugin`? I think it's currently stable after the PRs a couple weeks ago

Reply:
the missing plugin error is because >44 (idr the actual number) it now requires you using an eslint@8 config which sources plugins differently.

Reply:
we werent able to publish the stable release before the last rotation ended, will try to do so relatively soon. but the latest pre-release can be considered an rc;
and mike is right, you will need to switch to eslint 8

Reply:
According to the ESLint docs, the flat config file support is feature complete but not turned on by default. In order to use the latest @itwin/eslint-plugin you have to use the new config format?

Reply:
the way we deliver rules and plugins are different where i think it would be required to update to eslint 8

Reply:
We're using eslint@8 just don't have a flat config file.

Reply:
Will talk to the developer about why when we moved to eslint flat config support we no longer support eslint@7 in any form, but the new export type is designed explicitly for flat config consumption

Reply:
Our eslint config is in package.json. However, while we are using eslint 8 as Todd indicated, if I update to @itwin/eslint-plugin@4.0.0-dev.48 I get a missing plugin error. Our package.json eslintConfig section looks like this:

```json
  "eslintConfig": {
    "plugins": [
      "@itwin"
    ],
    "extends": "plugin:@itwin/itwinjs-recommended",
    "rules": {
      "nonblock-statement-body-position": "off"
    }
  },
```

The error message is this:

```
Oops! Something went wrong! :(

ESLint: 8.37.0

ESLint couldn't find the config "plugin:@itwin/itwinjs-recommended" to extend from. Please check that the name of the config is correct.

The config "plugin:@itwin/itwinjs-recommended" was referenced from the config file in "/Users/travis/Dev/itwin/mobile-sdk-core/package.json".
```

Reply:
Yes I have run into this. It is because of the way configs are exposed by the new plugin, only works with eslint flat config approach

Reply:
@tcobbs-bentley We can try configuring eslint as described in the plugin's readme: https://github.com/iTwin/eslint-plugin#usage

Reply:
Thanks. I agree, but I don't think that needs to be in this PR (or the other 2 PRs that use @itwin/eslint-plugin).

Reply:
> @tcobbs-bentley We can try configuring eslint as described in the plugin's readme: https://github.com/iTwin/eslint-plugin#usage

I did a quick test of this with the dev-48 release, and it does work, but there are a bunch of new eslint errors that I will have to address. So that definitely needs to be done in separate PRs.